_posts/2019-01-08-Golang.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: 'Golang Interface for TVM Runtime'
+author: Siva
+date: 2019-01-08
+---
+
+## Introduction
+
+TVM is an open deep learning compiler stack to compile various deep learning models from different
+frameworks to CPU, GPU or specialized accelerators.  TVM supports model compilation from a wide range
+of front ends like Tensorflow, Onnx, Keras, Mxnet, Darknet, CoreML and Caffe2. TVM compiled modules
+can be deployed on backends like LLVM (Javascript or WASM, AMD GPU, ARM or X86), NVidia GPU (CUDA),
+OpenCL and Metal.
+
+TVM supports runtime bindings for programming languages like Javascript, Java, Python, C++... and now Golang.
+With a wide range of frontend, backend and runtime bindings, TVM enables developers to integrate and
+deploy deep learning models from a variety of frameworks to a choice of hardware via many programming languages.
+
+TVM import and compilation process generates a graph JSON, a module and a params. Any application that
+integrates the TVM runtime can load these compiled module and infer. Detailed tutorial of module
```

Comment:
load these compiled module[s] and [perform inference]. [A] detailed tutorial of module